Assets/SEE/UI/Window/PropertyWindow/PropertyWindow.cs

```diff
@@ -0,0 +1,845 @@
+
+                if (activate)
+                {
+                    AnimateIn(go);
+                }
+                else
+                {
+                    if (GameFinder.FindChild(go, "Expand Icon") != null)
+                    {
+                        RotateExpandIcon(go, false);
+                    }
+                }
+
+                if (expandedItems.Contains(go.name))
+                {
+                    SetActive(GetDictOfGroup(go.name), activate);
+                    RotateExpandIcon(go, activate);
+                }
+            }
+            return;
+
+            /// Expands the game object by animating its scale.
+            static void AnimateIn(GameObject go)
+            {
+                go.transform.localScale = new Vector3(1, 0, 1);
+                go.transform.DOScaleY(1, duration: 0.5f);
+            }
+        }
+
+        /// <summary>
+        /// Creates the property window.
+        /// </summary>
+        public void CreateUIInstance()
+        {
+            /// Instantiate PropertyWindow
+            GameObject propertyWindow = PrefabInstantiator.InstantiatePrefab(WindowPrefab, Window.transform.Find("Content"), false);
+            propertyWindow.name = "Property Window";
+
+            items = (RectTransform)propertyWindow.transform.Find("Content/Items");
+            searchField = propertyWindow.transform.Find("Search/SearchField").gameObject.MustGetComponent<TMP_InputField>();
+
+            searchField.onSelect.AddListener(_ => SEEInput.KeyboardShortcutsEnabled = false);
+            searchField.onDeselect.AddListener(_ => SEEInput.KeyboardShortcutsEnabled = true);
+
+            ButtonManagerBasic filterButton = propertyWindow.transform.Find("Search/Filter").gameObject.MustGetComponent<ButtonManagerBasic>();
+            ButtonManagerBasic sortButton = propertyWindow.transform.Find("Search/Sort").gameObject.MustGetComponent<ButtonManagerBasic>();
+            ButtonManagerBasic groupButton = propertyWindow.transform.Find("Search/Group").gameObject.MustGetComponent<ButtonManagerBasic>();
+            PopupMenu.PopupMenu popupMenu = gameObject.AddComponent<PopupMenu.PopupMenu>();
+            UnityEvent rebuild = new();
+            rebuild.AddListener(() => Rebuild());
+            contextMenu = new PropertyWindowContextMenu(popupMenu, rebuild, filterButton, sortButton, groupButton);
+
+            CreateItems();
+            searchField.onValueChanged.AddListener(searchQuery => ActivateMatches(searchQuery));
+        }
+
+        /// <summary>
+        /// Rebuilds the window.
+        /// </summary>
+        private void Rebuild()
+        {
+            ClearItems();
+            CreateItems();
+        }
+
+        /// <summary>
+        /// Clears the property window of all items.
+        /// </summary>
+        private void ClearItems()
+        {
+            for (int i = 0; i < items.childCount;)
+            {
+                DestroyImmediate(items.GetChild(i).gameObject);
```

Comment:
I don't think you should use `DestroyImmediate` here, since this is not editor code. See earlier comment.

Reply:
Destroyer.Destroy destroys the object in the next frame. However, immediate destruction of the elements is required here.